### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by Determancer**
Always sync the latest progression.

### CODE REVIEW

**Comment by Determancer** [comment]
The Spell Types were not yet localized (instant and charge)
@Shinoow

File: src/main/resources/assets/abyssalcraft/lang/zh_cn.lang [hunk 1]
```diff
@@ -289,17 +289,17 @@ necronomicon.information.pantheon=万神殿
...
-necronomicon.spell.name=Spell name
-necronomicon.spell.pe=PE per cast
-necronomicon.spell.type=Spell type
-necronomicon.height=Height: %s blocks
-necronomicon.width=Width: %s blocks
-necronomicon.depth=Depth: %s blocks
-necronomicon.rangeamplifier=Range amplifier
-necronomicon.durationamplifier=Duration amplifier
-necronomicon.poweramplifier=Power amplifier
+necronomicon.createspells=咒语铭刻
```
Comment (Determancer) [inline src/main/resources/assets/abyssalcraft/lang/zh_cn.lang:292]:
```suggestion
necronomicon.createspells=雕绘咒文
```

File: src/main/resources/assets/abyssalcraft/lang/zh_cn.lang [hunk 2]
```diff
@@ -311,7 +311,7 @@ necronomicon.text.necronomicon.2=死灵之书是深渊之书中的一章，由
 necronomicon.text.necronomicon.3=深渊之书内共有4种死灵之书：死灵之书，深渊荒原死灵之书，恐惧之地死灵之书和奥穆索死灵之书，每一本都与一个维度有关。
 necronomicon.text.necronomicon.4=要想解锁这些知识，必须要旅行到上述的世界并且通过该世界里的兽皮来赐予死灵之书解锁的力量。深渊之书囊括了额外的只有扎哈尔知晓的知识，这是人类不应获得的知识。
 
-necronomicon.text.abyssalnomicon.1=这是深渊之书，是由深渊的守门者扎哈尔所写的书。这本书囊括了四本死灵之书的知识，毕竟上它们实际上只是这本书的章节。同样这里有大量的在其它地方没有提及的知识。
+necronomicon.text.abyssalnomicon.1=这是深渊之书，是由深渊守门者扎哈尔所写的书。这本书囊括了四本死灵之书的知识，毕竟上它们实际上只是这本书的章节。同样这里有大量的在其它地方没有提及的知识。
```
Comment (Determancer) [inline src/main/resources/assets/abyssalcraft/lang/zh_cn.lang:314]:
```suggestion
necronomicon.text.abyssalnomicon.1=这是深渊之书，是由深渊守门者扎哈尔所写的书。这本书囊括了四本死灵之书的知识，毕竟实际上后者只是这本书的章节。同样这里有大量的在其它地方没有提及的知识。
```

File: src/main/resources/assets/abyssalcraft/lang/zh_cn.lang [hunk 3]
```diff
@@ -486,7 +486,7 @@ necronomicon.text.dreadlands.progression.1=在你到达了恐惧之地后，你
 necronomicon.text.dreadlands.progression.2=当你有2个祭坛零件，你要将它们放在恐惧之地山脉的Y=40的高度并且右击它。做完后你就能进入到查伽洛斯的巢穴。在打败恐惧守卫并杀死查伽洛斯之后，你会获得一把恐惧疫染的门钥匙，随同着战利品。这把钥匙需要嬗变才能变成可用的钥匙。
 necronomicon.text.dreadlands.progression.3=通过前面提及的嬗变器（合成表在深渊荒原特殊材料章节）你可以将恐惧疫染的门钥匙嬗变成查伽洛斯的拉莱耶门钥匙，给予你前往奥穆索的途径。升级你的死灵之书到奥穆索死灵之书需要8个奥穆索兽皮。
 
-necronomicon.text.omothol.progression.1=现在你已经到了奥穆索，到了与深渊的守门者扎哈尔战斗的时候了。扎哈尔神殿坐落于奥穆索的中心（坐标 X=4，Y=53，Z=7）。击败守门者之后，你可以解锁死灵之书上的所有知识。
+necronomicon.text.omothol.progression.1=现在你已经到了奥穆索，到了与深渊守门者扎哈尔战斗的时候了。扎哈尔神殿坐落于奥穆索的中心（坐标 X=4，Y=53，Z=7）。击败守门者之后，你可以解锁死灵之书上的所有知识。
```
Comment (Determancer) [inline src/main/resources/assets/abyssalcraft/lang/zh_cn.lang:489]:
```suggestion
necronomicon.text.omothol.progression.1=现在你已经抵达奥穆索，是时候与深渊守门者扎哈尔战斗了。扎哈尔神殿坐落于奥穆索的中心（坐标 X=4，Y=53，Z=7）。击败守门者之后，你可以解锁死灵之书上的所有知识。
```

**Comment by Shinoow** [comment]
Two more fields to localize, then I can merge this and include the changes in the next release.

**Comment by Determancer** [comment]
It's done :P

**Pull request merged**